content/Watsonx/CodeAssistant/101.md

```diff
@@ -0,0 +1,78 @@
+---
+title: '101: Ansible Lightspeed Environment Setup'
+timeToComplete: 15
+updated: 2023-09-13
+---
+
+# 101: Ansible Lightspeed environment setup in VS Code
+
+## Goal
+
+The goal of this lab is to install the Ansible Lightspeed extension and enable and authenticate the extension in VS Code. 
+
+## Install Ansible extension in VS Code
+
+1. Open the VS Code application.
+2. Click the **Extensions** icon on the left sidebar.
+
+    ![](./images/settings-icon.png)
+3. Search for **Ansible** in the search box and then click the **Install** button.
+
+    ![](./images/search-for-ansible.png)
+
+## Enable the Ansible Lightspeed extension
+
+1. After installing the Ansible extension, click on the little gear icon and select the **Extension Setting** option.
+
+    ![](./images/gear-icon.png)
+2. In the Search settings bar, type **lightspeed**
+3. If they are not already selected, check all of the following boxes:
+ - Ansible > Lightspeed: **Enabled**
+ - Ansible > Lightspeed > Suggestions: **Enabled**.
+
+    ![](./images/enable-lightspeed.png)
+
+**Note:** For the purpose of this demo, we will disable Ansible Lint and Ansible Execution Environment.
+
+1. In the Search settings bar, type **ansible lint**. If it is not already unselected, *uncheck* the **Ansible > Validation > Lint: **Enabled**** box.
+
+    ![](./images/disable-ansible-lint.png)
+
+2. In the Search settings bar, type **ansible execution**. If it is not already unselected, *uncheck* the **Ansible > Execution Environment: **Enabled**** box.
+
+    ![](./images/disable-ansible-exec.png)
+
+## Authenticate Ansible Lightspeed with GitHub
+
+1. Open the Ansible extension icon on the left-hand side of the Visual Studio Code interface and click **Connect**.
+
+    ![](./images/connect-ansible.png)
+
+2. Follow the prompts to sign in to Ansible Lightspeed with IBM watsonx Code Assistant using your GitHub credentials.
+
+    a. Click on the **Allow** button.
+
+    ![](./images/ansible-sign-in.png)
+
+    b. Click on the **Open** button.
+
+    ![](./images/external-website.png)
+
+    c. Click on the **Log in with GitHub** button.
+
+    ![](./images/github-login.png)
+
+    d. Click on the **Authorize** button
+
+    ![](./images/authorize-ansible.png)
+
+    e. Click on the **Authorize ansible** button
+
+    ![](./images/authorize-ansible-github.png)
+
+    f. Upon successful completion of the login flow, you should see a **Logged in as your GitHub id** message in the Ansible side bar like the one below:
```

Comment:
Not sure if it's important to show screenshots for this, but I was presented with the following pop ups, which I think are covered as part of the "completion of the login flow".   You may want to include something to the effect of "You may be prompted to allow access to VS Code from your browser and vice versa.  Make sure you allow this."
<img width="712" alt="Screen Shot 2023-09-17 at 2 49 05 PM" src="https://github.com/ibm-build-lab/VAD-VAR-Workshop/assets/7364498/1d24bd71-6b10-495b-a079-ccda372ef2b7">
<img width="1368" alt="Screen Shot 2023-09-17 at 2 49 30 PM" src="https://github.com/ibm-build-lab/VAD-VAR-Workshop/assets/7364498/6cbba92a-d340-405d-b4e4-95ab00f17c18">